style.css

```diff
@@ -1413,5 +1405,4 @@ label {
   .boton {
     margin-top: 56px;
   }
-  
-}
\ No newline at end of file
+}
```

Comment:
kindly implement the following design changes to have a better UI and user interaction:

1. the logo is on the extreme left which is too close to the margin and not similar to the design given, could you fix this?
2. the font styling for the navigation is different eith the figma design given, kindly always make yoiur design close to the design given 😄 

your design :
![image](https://github.com/santitoot/desktop/assets/20537712/083a54ff-16db-4916-bc1e-60bb9a0cdd39)

**ABOUT SECTION**
3.  There are 2 blocks The left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right. kindly make the languages section take up the required `60%` of this section as it is in the figma design given to have a better UI and also work on the font color for that section
Your design :

![image](https://github.com/santitoot/desktop/assets/20537712/af2d481a-f7aa-4fa9-9798-7a3820493f15)

CONTACT SECTION
4. KINDLY MAKE THE BACKGROUND IMAGES TO BE SIMILAR TOTHE FIGMA TEMPLATE GIVEN  

your design :
![image](https://github.com/santitoot/desktop/assets/20537712/94234075-0ec6-47d7-aafb-ab350bf487b2)